blackdevelopa/iReporter
# #162584405 Get all red-flags endpoint

### What does this PR do?
This PR gets all red-flag endpoint from the database

### Description of task to be completed?
* The get all red-flag endpoint will return a status code of 200 and an array of data listing out all saved red-flag entries.
* The database is stored remotely on ElephantSQL

### How should this be manually tested?
* Git clone to localhost
* Launch Postman to localhost:3000/api/v1/red-flags
* Ensure it's on the GET method and click send.

### Any background context you want to provide?
There are certain limitations to the get all endpoint. Here's what you should know:
1. This feature didn't consider extreme cases for images, videos. This will be considered and raised later.
2. This PR does not take into consideration other endpoints.


### What are the relevant pivotal tracker stories?
#162584405


![screen shot 2018-12-12 at 4 30 53 am](https://user-images.githubusercontent.com/29962968/49845390-d2fdcb00-fdc6-11e8-86f8-a370486b7a51.png)
